src/service/genesis_gen.h

```diff
@@ -420,6 +420,15 @@ namespace ccf
         {.status = CodeStatus::ALLOWED_TO_JOIN, .platform = platform});
     }
 
+    void trust_executor_code_id(
+      const CodeDigest& executor_code_id, const QuoteFormat& platform)
+    {
+      auto executor_codeid = tx.rw(tables.executor_code_ids);
+      executor_codeid->put(
+        executor_code_id,
+        {.status = CodeStatus::ALLOWED_TO_JOIN, .platform = platform});
+    }
```

Comment:
This shouldn't be necessary as all modifications to this new table should be done in JS, via governance.

Reply:
Done